docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,77 @@
+
+## Key concepts
+
+While a detailed list and usage of the new features can be found in the [migration guide](https://aka.ms/js-track2-migration-guide) and the [quick start guide](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/next-generation-quickstart.md) can be used for first time users, below is an overview of the major changes in the new libraries
+
+### Authentication
+
+Previously, the Azure management libraries required credential objects created via the packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` that used ADAL which reaches end of support in Jun 2022. The new Azure management libraries require credential objects from the new `@azure/identity` package that makes use of the [MSAL](https://docs.microsoft.com/azure/active-directory/develop/msal-overview) and have many more authentication options to support newer scenarios.
+
+- For the different authentication scenarios, see the [Azure Identity examples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md)
+- Refer to the [migration guide to @azure/identity](https://github.com/Azure/ms-rest-nodeauth/blob/master/migrate-to-identity-v2.md) to update your existing application.
+
+### Promises vs callbacks
+
+To help with code readability, better handling of deferred and asynchronous computations and better error handling, our new guidelines encourage the use of promises over callbacks. Therefore, the overloads that took callbacks have been dropped in favor of promises.
+
+### PagedAsyncIterableIterator interface
+
+List operations now return a `PagedAsyncIterableIterator` interface that can be used to iterate over the results. This is a better user experience than before where a new method call had to made to access the next page. The [List Operations](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#list-operations) section in the migration guide will give you more details on it.
+
+### ES6 Support
+
+The SDKs have been changed to support ECMAScript 2015 (ES6) and beyond
+
+### Long Running Operations
+
+The LROPoller return object has been modified to PollerLike object. The `LROPoller` return object has been modified to `PollerLike` object. The [Long Running Operations](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#long-running-operations) section in the migration guide will give you more details on it.
```

Comment:
Could you use the aka.ms link   https://aka.ms/js-track2-migration-guide#long-running-operations

Reply:
Done